### PR TITLE
Rubocop 0.55

### PR DESCRIPTION
**Pull request by DirtyF**
Fix CI in #60

### CODE REVIEW

File: lib/jekyll-mentions.rb [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 module Jekyll
   class Mentions
-    GITHUB_DOT_COM = "https://github.com".freeze
-    BODY_START_TAG = "<body".freeze
+    GITHUB_DOT_COM = "https://github.com"
+    BODY_START_TAG = "<body"
```
Comment (pathawks) [inline lib/jekyll-mentions.rb]: 🎉🎉🎉

File: lib/jekyll-mentions.rb [hunk 2]
```diff
@@ -84,7 +84,7 @@ def mention_base(config = {})
       # Returns true if the doc is written & is HTML.
       def mentionable?(doc)
         (doc.is_a?(Jekyll::Page) || doc.write?) &&
-          doc.output_ext == ".html" || (doc.permalink && doc.permalink.end_with?("/"))
+          doc.output_ext == ".html" || (doc.permalink&.end_with?("/"))
```
Comment (pathawks) [inline lib/jekyll-mentions.rb]: I _really_ like this 👍🏼

**Comment by pathawks** [comment]
I suppose we should update the `gemspec` to require Ruby 2.3?

**Comment by DirtyF** [comment]
@jekyllbot: merge + minor

**Pull request merged**